### DISCUSSION TITLE
to_latex function

### DISCUSSION THREAD

**Discussion by paytonward6** [wooorm/markdown-rs#53] [Ideas]
Been toying around transpiling Markdown to LaTeX in Rust and ran across your project a few days ago. Would you ever be interested in implementing a `to_latex` function? I've done some testing of using your `to_html` code and replacing some of the HTML tags with their LaTeX equivalent and found it could be promising.  I would be more than willing to help contribute but just wanted to shoot the idea across before spending too much time on it

**Comment by wooorm** [comment]
Hi! I don’t plan on adding other functions like `to_html` here: `to_mdast` is the place where all of that should happen after. The only exception is `to_html` because that’s the most common case of using markdown. I recommend to use `to_mdast` and map one AST to another AST (your own latex AST), and then serialize that AST as a string of latex!
